### PR TITLE
JSAAL-17 Add investments value card

### PR DESCRIPTION
**Pull request by FairyWind**

### CODE REVIEW

File: src/controller/investments.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const userInvestments = require('../data/data');
+
+const investments = express.Router();
+investments.use(express.json());
+
+investments.get('/user/:uid', (req, res) => {
+  const { uid } = req.params;
+  const userData = userInvestments.find((data) => data.user_id === uid);
+  if (userData) {
+    return res.status(200).json(userData);
```
Comment (ajaxjiang96) [inline src/controller/investments.js]: You don't need to specify status code here and you can just use `res.send`
```suggestion
    return res.send(userData);
```

File: src/controller/investments.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  }
+  return res.status(404).send('User not found.');
```
Comment (ajaxjiang96) [inline src/controller/investments.js]:
```suggestion
  return res.sendStatus(404);
```

**Review by ajaxjiang96 — CHANGES_REQUESTED** [review]

**Review by ajaxjiang96 — APPROVED** [review]

File: src/controller/investments.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const userInvestments = require('../data/data');
+
+const investments = express.Router();
+investments.use(express.json());
```
Comment (ikarasz) [inline src/controller/investments.js]: json middleware is not needed, it is for parsinf request body

File: src/controller/investments.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+investments.get('/user/:uid', (req, res) => {
+  const { uid } = req.params;
+  const userData = userInvestments.find((data) => data.user_id === uid);
```
Comment (ikarasz) [inline src/controller/investments.js]: please use camelCase

**Review by ikarasz — CHANGES_REQUESTED** [review]

**Review by ikarasz — APPROVED** [review]

**Pull request merged**